src/test/scala/impl/group1/ExternalStabilityTest.scala

```diff
@@ -0,0 +1,169 @@
+package org.tud.sse.metrics
+package impl.group1
+
+import testutils.AnalysisTestUtils
+
+import org.scalatest.{FlatSpec, Matchers}
+import analysis.{MetricValue, MetricsResult}
+
+import java.io.File
+
+class ExternalStabilityTest extends FlatSpec with Matchers{
+
+  "The ExternalStabilityAnalysis" must "calculate valid results for multiple JARs" in {
+
+    val filesToTest = new File(getClass.getResource("/group1/commons-collections").getPath)
+    val analysisToTest = new ExternalStabilityAnalysis(filesToTest)
+
+    val appConfig = new ApplicationConfiguration(inputFilePath = "", treatFilesAsLibrary = true, outFileOption = None,
+      opalLoggingEnabled = false, batchModeEnabled = Some(false), excludedAnalysesNames = List(), includedAnalysesNames = List(),
+      excludeJreClasses = true, additionalClassesDir = None, loadAdditionalClassesAsInterface = false)
+
+    val optionsMap : Map[Symbol,Any]  = Map(Symbol("es_red") -> "es_red", Symbol("es_rem")-> "es_rem")
+    val result  = AnalysisTestUtils.runMultiFileAnalysis(_ => analysisToTest,filesToTest,appConfig,optionsMap)
+
+    assert(result.size == 1)
+
+    val metricResult = result.head
+    assert(metricResult.success)
+    assert(metricResult.metricValues.nonEmpty)
+
+
+    val values = result.head.metricValues.toList
+
+    assert(diffOf0001(values(8)))
```

Comment:
Dieser Test schlägt bei mir fehl. Grund wird vermutlich sein, dass die Reihenfolge der Ergebnisse in `values` bei mir eine andere ist, und darum der Zugriff mit festem Index (hier `8`) auf das "falsche" Ergebnis zugreift. Leider geben Methoden wie `toList` oder `appendAll` keine Garantien was die Reihenfolge angeht, darum können da auf verschiedenen Maschienen durchaus verschiedene Reihenfolgen rauskommen. 

Hier müsste sich einmal eine andere Teststrategie überlegt werden. Ich würde gucken dass man mit `values.find(v => v.entityIdent.contains(<gewünschter string>)` das passende Element findet, und dann gucken ob der Wert passt. @PKTUDO